src/main/scala/org/camunda/package.scala

```diff
@@ -189,17 +189,20 @@ package object feel {
     .appendValue(MONTH_OF_YEAR, 2)
     .appendLiteral("-")
     .appendValue(DAY_OF_MONTH, 2)
-    .toFormatter();
+    .toFormatter()
+    .withResolverStyle(ResolverStyle.STRICT)
```

Comment:
👍 Interesting solution

💭 Sadly [the JavaDoc](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/format/ResolverStyle.html#STRICT) is not explicit with examples. Still, your tests cover the interesting edge cases that we care about so that's alright.

Reply:
I found the trick with the resolver style when I checked the default behavior of `LocalDate.parse()`, which fails if the date is invalid.